Sources/Commands/SwiftTool.swift

```diff
@@ -769,6 +753,10 @@ public class SwiftTool {
             return self._hostToolchain
         }
 
+        // These flags are for host platforms, so clear them
```

Comment:
We need to clear them because extra flags of host destination include `-I` option which points XCTest.swiftmodule dir, and the compiler will discover host platform's XCTest module.